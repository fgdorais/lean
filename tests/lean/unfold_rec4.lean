open nat
definition f [unfold 1] (n : ℕ) : ℕ :=
by induction n with n fn; exact zero; exact succ (succ fn)

example (n : ℕ) : f (succ (succ n)) = sorry :=
begin
  unfold f,
end
