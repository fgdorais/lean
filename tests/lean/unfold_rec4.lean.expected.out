unfold_rec4.lean:8:0: error: 1 unsolved subgoal
n : ℕ
⊢ succ (succ (succ (succ (f n)))) = sorry
unfold_rec4.lean:8:0: error: failed to add declaration 'example' to environment, value has metavariables
remark: set 'formatter.hide_full_terms' to false to see the complete term
  ?M_1
